integration/harness-e2e-cli/package.json

```diff
@@ -8,35 +8,35 @@
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
     "run-e2e-specs": "node --loader ts-node/esm node_modules/jasmine/bin/jasmine --config=e2e/jasmine.json",
-    "wait-and-run-e2e": "wait-on http://localhost:4200 && yarn run-e2e-specs",
-    "e2e": "concurrently -s first -k 'ng serve' 'yarn wait-and-run-e2e'"
+    "wait-and-run-e2e": "wait-on http://localhost:4200 && pnpm run-e2e-specs",
+    "e2e": "concurrently -s first -k 'pnpm ng serve' 'pnpm wait-and-run-e2e'"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "file:../../node_modules/@angular/animations",
-    "@angular/cdk": "file:../../dist/releases/cdk",
-    "@angular/common": "file:../../node_modules/@angular/common",
-    "@angular/compiler": "file:../../node_modules/@angular/compiler",
-    "@angular/core": "file:../../node_modules/@angular/core",
-    "@angular/forms": "file:../../node_modules/@angular/forms",
-    "@angular/material": "file:../../dist/releases/material",
-    "@angular/platform-browser": "file:../../node_modules/@angular/platform-browser",
-    "@angular/router": "file:../../node_modules/@angular/router",
+    "@angular/animations": "20.0.0-next.5",
+    "@angular/cdk": "next",
```

Comment:
NIT: can this be a specific version?, Same for `@angular/material`

Reply:
I would want this to be `next` because those are in practice replaced at runtime with the `HEAD` packages. Happy for any other suggestion